testlib/src/test/java/com/diffplug/spotless/generic/LicenseHeaderStepTest.java

```diff
@@ -46,6 +54,26 @@ public void fromFile() throws Throwable {
 		assertOnResources(step, KEY_FILE_NOTAPPLIED, KEY_FILE_APPLIED);
 	}
 
+	@Test
+	public void should_apply_license_containing_YEAR_variable() throws Throwable {
+		FormatterStep step = LicenseHeaderStep.createFromFile(createTestFile(KEY_LICENSE_WITH_YEAR), StandardCharsets.UTF_8, LICENSE_HEADER_DELIMITER);
+
+		StepHarness.forStep(step)
+				.test(getTestResource(KEY_FILE_WITHOUT_LICENSE), getFileContentWithYEAR(currentYear()))
+				.testUnaffected(getFileContentWithYEAR(currentYear()))
+				.testUnaffected(getFileContentWithYEAR("2003"))
+				.testUnaffected(getFileContentWithYEAR("1990-2015"))
+				.test(getFileContentWithYEAR("not a year"), getFileContentWithYEAR(currentYear()));
+	}
+
+	private String getFileContentWithYEAR(String year) throws IOException {
```

Comment:
I think it makes more sense for the "YEAR" in this method name to be replaced with "YearToken", to be consistent with `LicenseHeaderStep.java` and Java practices in general.

Reply:
renamed. but its more really the content of the `$YEAR` than the token itself